app/components/calls_to_action/promo/right_section.html.erb

```diff
@@ -1,11 +1,10 @@
 <div class="promo__right">
-  <h2 class="heading-l">
-    <% if caption %>
-      <%= tag.div(caption, class: "caption-l caption--purple caption--bold") %>
-    <% end %>
-
-    <%= tag.div(heading) %>
-  </h2>
+  <%= render Content::HeadingComponent.new(
```

Comment:
@gazcarr These promo h2s and captions are now a bigger font size - just checking this is as expected from your perspective (fine with me either way)

eg h2 and caption: https://review-get-into-teaching-app-2975.london.cloudapps.digital/non-uk-teachers/international-qualified-teacher-status

eg longer h2: https://review-get-into-teaching-app-2975.london.cloudapps.digital/train-to-be-a-teacher/get-school-experience

Reply:
the promo H2's (below the captions) should be roughly 36px @ethax-ross